### DISCUSSION TITLE
x86 version?

### DISCUSSION THREAD

**Discussion by jacob755** [bisq-network/bisq#5793] [Ideas]
hi
i need this app to run windows 32 bit (x86)
please build it

thank you

**Comment by m52go** [comment]
Java 10 dropped support for 32-bit builds. As a result, as of some time around v0.9.1, 32-bit builds have no longer been shipped.

https://github.com/bisq-network/bisq/releases/tag/v0.9.1

**Reply by jacob755** [reply]
thank you
also googled found this
https://stackoverflow.com/questions/49976684/java-10-and-following-on-32-bit-systems
https://adoptopenjdk.net/

**Reply by m52go** [reply]
Maybe a developer can add more perspective but my gut feeling is that resources to develop and maintain 32-bit Windows builds will be hard to allocate. Especially when there is very little demand for it but a whole lot of demand for other architectures like ARM.
